addons/account/models/chart_template.py

```diff
@@ -259,7 +259,7 @@ def tax_template_changed(tax, template):
                     if rec:
                         for fname in x2manyfields:
                             for i, (line, vals) in enumerate(zip(rec[fname], values[fname])):
-                                values[fname][i] = Command.update(line.id, vals[2])
+                                values[fname][i] = Command.update(line.id, vals[2]) if isinstance(vals[2], dict) else vals
```

Comment:
The type of `dict` shouldn't be the driver for what we are doing here, it should be the ORM command.
The purpose of this is to transform `CREATE` commands into `UPDATE` commands when reloading a template, the issue is that the command is no a `CREATE`